cmd/snap-update-ns/bootstrap.c

```diff
@@ -327,7 +327,7 @@ int validate_instance_name(const char *instance_name)
 // process_arguments parses given a command line
 // argc and argv are defined as for the main() function
 void process_arguments(int argc, char *const *argv, const char **snap_name_out,
-		       bool * should_setns_out, bool * process_user_fstab)
+		       bool * should_setns_out, bool * process_user_fstab, int * uid_out)
```

Comment:
This should probably be `uid_t`

Reply:
Note: I switched to unsigned long. It should be sufficient and explicit (I can format it).